backend/event/apps.py

```diff
@@ -4,4 +4,4 @@
 class EventConfig(AppConfig):
     default_auto_field = 'django.db.models.BigAutoField'
     name = 'event'
-    verbose_name = 'epistolary episodes and actions'
+    verbose_name = 'events and actions'
```

Comment:
I think this change is just for consistency, but in general, I think "episode/action" is clearer terminology than "event/action".

Reply:
My reasoning was that the term 'episodes' is otherwise not (yet) used, so it didn't make much sense to include it here. I agree that 'events' is a bit vague, so if you have any suggestions for terms that encapsulate both world and epistolary events, let me know!